modules/electrophysiology_browser/jsx/electrophysiologySessionView.js

```diff
@@ -192,10 +192,10 @@ class ElectrophysiologySessionView extends Component {
       }
       return resp.json();
     })
-        .then((data) => {
+      .then((data) => {
           const database = data.database.map((dbEntry) => ({
             ...dbEntry,
-            // EEG Visualisation urls
+            // EEG Visualisation parameters
```

Comment:
```suggestion
            // EEG Visualization parameters
```
making an irrelevant typo correction just to show jefferson how suggest works